gradle/wrapper/gradle-wrapper.properties

```
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.1-bin.zip
```

Since we're updating, why not 7.4.2 (latest as of Jun 7th)?

```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-bin.zip
```

---

Yeah, I kept in par with Logstash ones, but updating to latest seems good